MusicApp.Identity.Application/JwtManager.cs

```diff
@@ -0,0 +1,59 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.IdentityModel.Tokens;
+using MusicApp.Identity.Domain.Entities;
+using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using System.Security.Cryptography;
+
+namespace MusicApp.Identity.Application;
+
+public static class JwtManager
```

Comment:
it is better to do class like service, register it via ioc and then inject where it is necessary. In case of your code testing is very hard for this methods